SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1441,6 +1442,27 @@ void USpatialReceiver::ResolveObjectReferences(FRepLayout& RepLayout, UObject* R
 
 				ObjectProperty->SetObjectPropertyValue(Data + AbsOffset, SinglePropObject);
 			}
+			else if (ObjectReferences.bFastArrayProp)
+			{
+				int64 CountBits = ObjectReferences.Buffer.Num() * 8;
+				TSet<FUnrealObjectRef> NewUnresolvedRefs;
+				FSpatialNetBitReader ValueDataReader(PackageMap, ObjectReferences.Buffer.GetData(), CountBits, NewUnresolvedRefs);
+
+				SpatialFastArrayNetSerializeCB SerializeCB(NetDriver);
+
+				FSpatialNetDeltaSerializeInfo Parms = FSpatialNetDeltaSerializeInfo::CreateReader(ValueDataReader, SerializeCB);
+
+				UStructProperty* ParentStruct = Cast<UStructProperty>(Property);
```

Comment:
Above we do a `check` on this property. Should we also `check` here or make this assignment an if.